nx.json

```diff
@@ -0,0 +1,39 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": [
+          "test",
+          "build"
+        ],
+        "parallel": 5
+      }
+    }
+  },
+  "namedInputs": {
+    "default": [
+      "{projectRoot}/**/*"
+    ],
+    "prod": [
+      "!{projectRoot}/**/*.test.tsx"
```

Comment:
Last comment (just reposting here for completeness) was about this line—should we make the extension more inclusive to other test files?

Reply:
We're going to change it to `"prod": ["!{projectRoot}/**/*.test.{tsx,ts,js,jsx}"]`.